recipe/build.sh

```diff
@@ -1,28 +1,24 @@
 #!/bin/bash
 
-export CFLAGS="-I$PREFIX/include -L$PREFIX/lib"
+set -e
 
-# As of Mac OS 10.8, X11 is no longer included by default
-# (See https://support.apple.com/en-us/HT201341 for the details).
-# Due to this change, we disable building X11 support for cairo on OS X by
-# default.
-export XWIN_ARGS=""
-if [ $(uname) == Darwin ]; then
-    export XWIN_ARGS="--disable-xlib -disable-xcb --disable-glitz"
-fi
+export CFLAGS="-I$PREFIX/include -L$PREFIX/lib"
 
-./configure \
-    --prefix="${PREFIX}" \
-    --enable-warnings \
-    --enable-ft \
-    --enable-ps \
-    --enable-pdf \
-    --enable-svg \
-    --disable-gtk-doc \
-    $XWIN_ARGS
+configure_args=(
+    --prefix="$PREFIX"
+    --enable-warnings
+    --enable-ft
+    --enable-ps
+    --enable-pdf
+    --enable-svg
+    --disable-gtk-doc
```

Comment:
It's just a better way to deal with them, in my opinion. You can extend the lists a little more cleanly if there are platform-specific changes to make ... which is what inspired me to edit them here, except the point of the commit was to remove the platform-specific changes!

The real benefit, though, is that it lets you have an editable list of arguments when some of them have internal spaces, like can easily happen with things along the lines of `--cflags="$CFLAGS"`. When you use `bash` lists you can extend the list of arguments *and* preserve the internal spaces correctly upon expansion with `"${list[@]}"`.